plugins/bananaslug/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+clear;
+
+echo 'build plugin "bananaslug"';
+
+# clean up old files
+rm build/*
```

Comment:
the first time, build is not a directory -- we would need to create if it didn't exist